pitop/camera/camera.py

```diff
@@ -187,6 +190,8 @@ def __process_camera_output(self):
             try:
                 self.__frame_handler.frame = self.__camera.get_frame()
                 self.__new_frame_event.set()
+                if callable(self.on_frame):
+                    self.on_frame(self.__frame_handler.frame)
                 self.__frame_handler.process()
             except Exception as e:
                 print(f"There was an error: {e}")
```

Comment:
I think we should improve this message, just came across it and it feels a bit scrappy. Maybe `Error in camera frame handler:` or just print the plain error.

Reply:
agreed, added separate try/except blocks to catch errors when running `on_frame` and another one for `FH.process()`